co-med-backend-services/config/default.toml

```diff
@@ -0,0 +1,6 @@
+[httpParameters]
+districtCode = 696
+url = "https://cdn-api.co-vin.in/api/v2/appointment/sessions/public/calendarByDistrict?"
```

Comment:
get.calendar.by.district.url

Reply:
changed it to get_calender_by_district_url